cli/pkg/kctrl/cmd/app/app.go

```diff
@@ -29,7 +28,29 @@ func isOwnedByPackageInstall(app *kcv1alpha1.App) bool {
 	return false
 }
 
-func getAppDescription(name string, namespace string) string {
-	description := fmt.Sprintf("app/%s (kappctrl.k14s.io/v1alpha1)  namespace: %s", name, namespace)
-	return description
+func appStatusString(app *kcv1alpha1.App) string {
+	if len(app.Status.Conditions) < 1 {
+		return ""
+	}
+	if app.Spec.Canceled {
+		return color.RedString("Camceled")
+	}
+	if app.Spec.Paused {
+		return color.YellowString("Paused")
+	}
+	for _, condition := range app.Status.Conditions {
+		switch condition.Type {
+		case kcv1alpha1.ReconcileFailed:
+			return color.RedString("Reconcile failed")
```

Comment:
lets use ValueFmt (https://github.com/vmware-tanzu/carvel-kapp/blob/develop/pkg/kapp/cmd/app/list.go#L139)

Reply:
This would only allow us to color the output red if the Error boolean is true right?
As of now we are also coloring paused status yellow, that won't be possible with ValueFmt I think.

I see there is the possibility of having a "Func" formatting the value but it looks like the contructor does not allow for it

Reply:
I was giving this a shot, but I think it does not fit the case well here since we are not picking up a value directly from the status (which we would assign to `ValueFmt.V`)
If the app is paused or cancelled, we show "Paused" or "Cancelled" instead of formatting the friendly error message that is "Canceled/Paused"

Reply:
> I was giving this a shot, but I think it does not fit the case well here since we are not picking up a value directly from the status (which we would assign to ValueFmt.V)

im not following.

ValueFmt takes arbitrary value. example: https://github.com/vmware-tanzu/carvel-kapp/blob/develop/pkg/kapp/cmd/app/list.go#L139-L142

V of ValueFmt could be anything include string value any way you decide to construct. there is no relationship to where the value comes from. whatever logic you have today, you can construct a string value and give it to ValueFmt.